src/components/NavBar.tsx

```diff
@@ -1,12 +1,14 @@
-import { styles } from './SDKContainer.styled';
+import { useSDKStyles } from './SDKContainer.styled';
 import { Image, TouchableOpacity, View } from 'react-native';
 
 type NavBarProps = {
   goBack: () => void;
   showCloseAlert: () => void;
 }
 
-const NavBar = ({goBack, showCloseAlert}: NavBarProps) => {
+export const NavBar = ({goBack, showCloseAlert}: NavBarProps) => {
```

Comment:
Is Navbar exported for clients to use? If it is, changing to named export from default one would be breaking if clients are importing it in their app.